mlrun/model_monitoring/db/tsdb/tdengine/schemas.py

```diff
@@ -125,16 +125,41 @@ def _get_subtables_query(
     @staticmethod
     def _get_records_query(
         table: str,
-        start: str,
-        end: str,
+        start: datetime,
+        end: datetime,
         columns_to_filter: list[str] = None,
         filter_query: str = "",
+        interval: str = "",
+        limit: int = 0,
+        agg: Optional[list] = None,
```

Comment:
```suggestion
        aggregations: Optional[list] = None,
```

Reply:
changed it `agg_func`, I think it's a good indication of aggregated functions. LMK if you think it's better to use `aggregations`.

Reply:
Maybe `agg_funcs` to hint for the multi-func option.